p2p/p2p-core/src/network_zones/clear.rs

```diff
@@ -45,26 +45,10 @@ pub struct ClearNetServerCfg {
 #[derive(Clone, Copy)]
 pub enum ClearNet {}
 
-const fn ip_v4(a: u8, b: u8, c: u8, d: u8, port: u16) -> SocketAddr {
-    SocketAddr::new(IpAddr::V4(Ipv4Addr::new(a, b, c, d)), port)
-}
-
 #[async_trait::async_trait]
 impl NetworkZone for ClearNet {
     const NAME: &'static str = "ClearNet";
 
-    const SEEDS: &'static [Self::Addr] = &[
-        ip_v4(176, 9, 0, 187, 18080),
-        ip_v4(88, 198, 163, 90, 18080),
-        ip_v4(66, 85, 74, 134, 18080),
-        ip_v4(51, 79, 173, 165, 18080),
-        ip_v4(192, 99, 8, 110, 18080),
-        ip_v4(37, 187, 74, 171, 18080),
-        ip_v4(77, 172, 183, 193, 18080),
-    ];
```

Comment:
Doesn't removing this mean users will have to manually provide seed nodes?

Will there eventually be default seed nodes added to the `P2PConfig`?

Reply:
> Doesn't removing this mean users will have to manually provide seed nodes?

Users as in devs using `cuprate-p2p`? yes.

Users as in people using `cuprated`? no.

> Will there eventually be default seed nodes added to the P2PConfig?

in #304 I am creating the config that will be exposed to `cuprated` users in there there will be default seed nodes